### PR TITLE
Custom m utils create

### PR DESCRIPTION
**Pull request by hpasapp**
- create `custom_m_utils.py` module, which provides utilities for creating an SRU using Linears or Sequential[Linear,Linear], in custom_m; and for converting an existing SRU to use linears or Sequential[Linear,Linear], without loss of already-trained parameters.
- create unit tests for `custom_m_utils.py`

### CODE REVIEW

File: sru/custom_m_utils.py [hunk 1]
```diff
@@ -0,0 +1,188 @@
+from torch import nn
```
Comment (taoleicn) [inline sru/custom_m_utils.py:1]: Can we rename the file to sth like `utils.py`? `custom_m` is not informative and doesnot sound like a great name to me.

File: sru/custom_m_utils.py [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+import logging
+from typing import List, Union
+from sru import SRU, SRUCell
+
+logger = logging.getLogger(__name__)
+
+
+def convert_sru_cell_to_linears(cell: SRUCell):
```
Comment (taoleicn) [inline sru/custom_m_utils.py:9]: Is this a public method that will be exposed to other libraries?
If so, mind if you add a doc string? also, these are inplace operations to the given `cell`.

File: sru/modules.py [hunk 3]
```diff
@@ -99,6 +99,7 @@ def __init__(self,
         self.input_size = input_size
         self.hidden_size = hidden_size  # hidden size per direction
         self.output_size = hidden_size * 2 if bidirectional else hidden_size
+        self.projection_size = 0
```
Comment (taoleicn) [inline sru/modules.py:102]: there is a duplicate line of this at line 118 / 119 of this file.

**Review by taoleicn — COMMENTED** [review]
I left some minor inline comments. A more major comment I have:

Instead of providing this utility, what are your thoughts on directly making `SRUCell` contain sub-modules `nn.Linear()` or `nn.Sequantial[Linear, Linear]` at construction? It will be a breaking change that we need to bump up a major version, but will make SRU readily prunable and quantizable by default.

**Comment by hpasapp** [comment]
I will close this, since replaced by https://github.com/asappresearch/sru/pull/153 and https://github.com/asappresearch/sru/pull/155
